docs/tutorials/alb-ingress.md

```diff
@@ -167,3 +167,45 @@ spec:
 The above Ingress object will result in the creation of an ALB with a dualstack
 interface. ExternalDNS will create both an A `echoserver.example.org` record and
 an AAAA record of the same name, that each are aliases for the same ALB.
+
+## Dualstack NLBs
+
+The ALB ingress controller satisifies service of "LoadBalancer" type with
```

Comment:
I think you mean NLB?

Reply:
well actually it is aws loadbalancer ingress controller. It satisfies ingress resource and service resource of type loadbalancer in presence of annotation. I will change just in case to avoid confusion.